src/sonarwhal-theme/layout/partials/rules-index.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="layout layout--halves">
+    {{#each categories}}
+    <div class="module placement">
+        <p class="title icon--categories icon--a11y">{{name}}</p>
+        <p class="placement__body">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean euismod bibendum laoreet. Proin gravida dolor
```

Comment:
I guess this is the description of the category that should be the same as in `rule-category.hbs`? I think we need a place where we can put all the information related to them:

```json
{
  "id": "categoryId",
  "name": "name", // So PWA is all capital
  "description": "description",
  "icon": "classIcon" // although maybe this one can be pulled from the id?
}
```

Maybe a series of helpers for this? `{{# categoryDescripition categoryId}}`,  `{{#categoryName categoryId}}` ?